internal/build/custom_builder.go

```diff
@@ -16,7 +16,7 @@ import (
 )
 
 type CustomBuilder interface {
-	Build(ctx context.Context, ref reference.Named, command string, expectedTag string, skipsLocalDocker bool) (reference.NamedTagged, error)
+	Build(ctx context.Context, ref reference.Named, tiltfilePath string, command string, expectedTag string, skipsLocalDocker bool) (reference.NamedTagged, error)
```

Comment:
i wonder if we should pass in the whole CustomBuild model? it seems weird to break it apart